v23.1/qlik.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Migrate and Replicate Data with Qlik Replicate
+summary: Use Qlik Replicate to migrate data to a CockroachDB cluster or replicate data to a secondary source.
+toc: true
+docs_area: migrate
+---
+
+[Qlik](https://www.qlik.com) offers a service called Qlik Replicate that you can use to do the following:
+
+- [Migrate data to CockroachDB](#migrate-and-replicate-data-to-cockroachdb) from an existing, publicly hosted database containing application data, such as PostgreSQL, MySQL, Oracle, or Microsoft SQL Server. 
+- [Replicate data to a secondary source](#replicate-data-from-cockroachdb-to-a-secondary-source) such as Kafka or cloud storage.
+
+This page describes the Qlik Replicate functionality at a high level. For detailed information, refer to the tutorial and documentation provided when [signing up for Qlik Replicate](https://www.qlik.com/us/products/qlik-replicate).
+
+## Before you begin
+
+Complete the following items before using Qlik Replicate:
+
+- Ensure you have a secure, publicly available CockroachDB cluster running the latest **{{ page.version.version }}** [production release](../releases/index.html), and have created a [SQL user](security-reference/authorization.html#sql-users) that you can use for your Qlik Replicate target endpoint.
```

Comment:
the copy_from_blah commands need to be changed as well.

Reply:
Added the two session var workarounds for 22.2 - this is what you meant, right?

Reply:
yes, they probs need to be in 23.1 too

Reply:
Ah right, added.